tests/database/00-simple.t

```diff
@@ -60,5 +60,10 @@ sqlite3 "${DB_FILE}" 'SELECT name, cycle, status FROM task_states ORDER BY name'
     >"${NAME}"
 cmp_ok "${TEST_SOURCE_DIR}/${TEST_NAME_BASE}/${NAME}" "${NAME}"
 
+NAME='select-linearization.out'
+sqlite3 "${DB_FILE}" 'SELECT namespace, linearization FROM linearization ORDER BY namespace' \
+    >"${NAME}"
+cmp_ok "${TEST_SOURCE_DIR}/${TEST_NAME_BASE}/${NAME}" "${NAME}"
+
```

Comment:
Maybe rebase this branch to master and protect the `sqlite3` call as documented here: https://github.com/cylc/cylc/blob/master/bin/cylc-test-battery#L77 (else we might forget...)